syft/serde.py

```diff
@@ -1332,17 +1354,18 @@ def _force_full_simplify(obj: object) -> object:
     MultiPointerTensor: [14, _simplify_multi_pointer_tensor],
     Plan: [15, _simplify_plan],
     VirtualWorker: [16, _simplify_worker],
-    GetNotPermittedError: [17, _simplify_GetNotPermittedError],
-    str: [18, _simplify_str],
-    pointers.ObjectWrapper: [20, _simplify_object_wrapper],
+    str: [17, _simplify_str],
+    pointers.ObjectWrapper: [19, _simplify_object_wrapper],
+    GetNotPermittedError: [20, _simplify_exception],
```

Comment:
This is very nit pick but can we make exceptions 17 instead? This will allow some compatibility across versions